src/features/posts/SinglePostPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { Link } from 'react-router-dom'
+
+import { selectPostById } from './postsSlice'
+
+export const SinglePostPage = ({ match }) => {
```

Comment:
Looks like you are missing ReactionButtons on the post page. This is one of the most important pieces of the tutorial that helps you see Redux in action.